smithy-aws-apigateway-openapi/src/main/java/software/amazon/smithy/aws/apigateway/openapi/AddDefaultRestConfigSettings.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.aws.apigateway.openapi;
+
+import java.util.List;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.openapi.OpenApiConfig;
+import software.amazon.smithy.utils.ListUtils;
+
+/**
+ * REST APIs require that name parameters for greedy labels are not suffixed with "+".
```

Comment:
This should probably link to their API docs showing that greedy labels are not suffixed for REST: https://docs.aws.amazon.com/apigateway/latest/developerguide/setup-http-integrations.html

Consider a "this is counter to the default, like for HTTP APIs, ..." that links to those docs too: https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-http.html

Reply:
Addressed in ee69c1c.